### ISSUE TITLE
update eslint-plugin-import to version 2.17.2 => Cannot find module 'babel-plugin-add-module-exports'

### ISSUE THREAD

**Issue by kopax** [benmosher/eslint-plugin-import#1336]
We are having this issues on all our repo when upgrading this plugin:

```bash

{ Error: Cannot find module 'babel-plugin-add-module-exports' from '/home/travis/build/bootstrap-styled/motion'
    at Function.module.exports [as sync] (/home/travis/build/bootstrap-styled/motion/node_modules/resolve/lib/sync.js:58:15)
    at resolveStandardizedName (/home/travis/build/bootstrap-styled/motion/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (/home/travis/build/bootstrap-styled/motion/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (/home/travis/build/bootstrap-styled/motion/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (/home/travis/build/bootstrap-styled/motion/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at items.map (/home/travis/build/bootstrap-styled/motion/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/home/travis/build/bootstrap-styled/motion/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (/home/travis/build/bootstrap-styled/motion/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at plugins (/home/travis/build/bootstrap-styled/motion/node_modules/@babel/core/lib/config/config-descriptors.js:40:19) code: 'MODULE_NOT_FOUND' }
```

Source: https://travis-ci.org/bootstrap-styled/motion/jobs/520976849

I've been looking and doesn't see any breaking change rencetly. any idea on what is happening here?

**Comment by benmosher** [comment]
looks unrelated based on the stack trace. I do not recall having or removing that package.

closing, but can reopen if you can show that this plugin is responsible

**Comment by kopax** [comment]
Hi @benmosher, I have opened this issue on `@babel/core`:

- https://github.com/babel/babel/issues/9867

I believe it is not [babel-merge](https://www.npmjs.com/package/babel-merge) and possibly not [resolve](https://github.com/browserify/resolve) because of their simple purpose.

You must be right it is perhaps not because of `eslint-plugin-import`, it's just greenkeeper hinting us in that direction. If it does not sound familiar, it's certainly not related. Thank you for the information.

**Comment by ljharb** [comment]
It’d an issue with n0w-unpublished v1.0.1 of add-module-exports

**Comment by kopax** [comment]
It is yes. We fixed it to 1.0.0. Thanks
